codes/kotlin/chapter_array_and_linkedlist/linked_list.kt

```diff
@@ -0,0 +1,80 @@
+/**
+ * File: linked_list.kt
+ * Created Time: 2024-01-25
+ * Author: curtishd(1023632660@qq.com)
+ */
+
+package chapter_array_and_linkedlist
+
+import utils.ListNode
+import utils.printLinkedList
+
+/* 在链表的节点 n0 之后插入节点p */
+fun insert(n0: ListNode?, p: ListNode?) {
+    val n1 = n0?.next
+    p?.next = n1
+    n0?.next = p
+}
+
+/* 删除链表的节点 n0 之后的首个节点 */
+fun remove(n0: ListNode?) {
+    val p = n0?.next
+    val n1 = p?.next
+    n0?.next = n1
+}
+
+/* 访问链表中索引为 index 的节点 */
+fun access(head: ListNode?, index: Int): ListNode? {
+    var h = head
+    for (i in 0..<index) {
+        h = h?.next
+    }
+    return h
+}
+
+/* 在链表中查找值为 target 的首个节点 */
```

Comment:
The code highlight style seems broken from this line. Is it a bug of the GitHub editor?

Reply:
I guess so.